packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -382,4 +397,213 @@ export class CustomRule extends RuleNew {
+  /** EC2 flow log */
+  public static readonly EC2_FLOW_LOG = new ResourceType('AWS::EC2::FlowLog');
+  /** EC2 VPC endpoint */
+  public static readonly EC2_VPC_ENDPOINT = new ResourceType('AWS::EC2::VPCEndpoint');
+  /** EC2 VPC endpoint service */
+  public static readonly EC2_VPC_ENDPOINT_SERVICE = new ResourceType('AWS::EC2::VPCEndpointService');
+  /** EC2 VPC peering connection */
+  public static readonly EC2_VPC_PEERING_CONNECTION = new ResourceType('AWS::EC2::VPCPeeringConnection');
+  /** Amazon ElasticSearch domain */
+  public static readonly ELASTICSEARCH_DOMAIN = new ResourceType('AWS::Elasticsearch::Domain');
+  /** Amazon QLDB ledger */
+  public static readonly QLDB_LEDGER = new ResourceType('AWS::QLDB::Ledger');
+  /** Amazon Redshift cluster */
+  public static readonly REDSHIFT_CLUSTER = new ResourceType('AWS::Redshift::Cluster');
+  /** Amazon Redshift cluster parameter group */
+  public static readonly REDSHIFT_CLUSTER_PARAMETER_GROUP = new ResourceType('AWS::Redshift::ClusterParameterGroup');
+  /** Amazon Redshift cluster security group */
+  public static readonly REDSHIFT_CLUSTER_SECURITY_GROUP = new ResourceType('AWS::Redshift::ClusterSecurityGroup');
+  /** Amazon Redshift cluster snapshot */
+  public static readonly REDSHIFT_CLUSTER_SNAPSHOT = new ResourceType('AWS::Redshift::ClusterSnapshot');
+  /** Amazon Redshift cluster subnet group */
+  public static readonly REDSHIFT_CLUSTER_SUBNET_GROUP = new ResourceType('AWS::Redshift::ClusterSubnetGroup');
+  /** Amazon Redshift event subscription */
+  public static readonly REDSHIFT_EVENT_SUBSCRIPTION = new ResourceType('AWS::Redshift::EventSubscription');
+  /** Amazon RDS database instance */
+  public static readonly RDS_DB_INSTANCE = new ResourceType('AWS::RDS::DBInstance');
+  /** Amazon RDS database security group */
+  public static readonly RDS_DB_SECURITY_GROUP = new ResourceType('AWS::RDS::DBSecurityGroup');
+  /** Amazon RDS database snapshot */
+  public static readonly RDS_DB_SNAPSHOT = new ResourceType('AWS::RDS::DBSnapshot');
+  /** Amazon RDS database subnet group */
+  public static readonly RDS_DB_SUBNET_GROUP = new ResourceType('AWS::RDS::DBSubnetGroup');
+  /** Amazon RDS event subscription */
+  public static readonly RDS_EVENT_SUBSCRIPTION = new ResourceType('AWS::RDS::EventSubscription');
+  /** Amazon RDS database cluster */
+  public static readonly RDS_DB_CLUSTER = new ResourceType('AWS::RDS::DBCluster');
+  /** Amazon RDS database cluster snapshot */
+  public static readonly RDS_DB_CLUSTER_SNAPSHOT = new ResourceType('AWS::RDS::DBClusterSnapshot');
+  /** Amazon SQS queue */
+  public static readonly SQS_QUEUE = new ResourceType('AWS::SQS::Queue');
+  /** Amazon SNS topic */
+  public static readonly SNS_TOPIC = new ResourceType('AWS::SNS::Topic');
+  /** Amazon S3 bucket */
+  public static readonly S3_BUCKET = new ResourceType('AWS::S3::Bucket');
+  /** Amazon S3 bucket where public access is blocked */
```

Comment:
This isn't actually specific to a bucket, it's whether there is an account-wide block on publicly-accessible buckets. See https://docs.aws.amazon.com/AmazonS3/latest/dev/access-control-block-public-access.html

```suggestion
  /** Amazon S3 account public access block */
```